src/Arcus.Scripting.ARM/Scripts/Inject-ArmContent.ps1

```diff
@@ -3,15 +3,28 @@
     
     ${ fileToInject.xml }
     ${ FileToInject=file.xml }
+    ${ FileToInject=c:\file.xml }
     ${ FileToInject = ".\Parent Directory\file.xml" }
+    ${ FileToInject = "c:\Parent Directory\file.xml" }
     ${ FileToInject = ".\Parent Directory\file.xml", EscapeJson, ReplaceSpecialChars }
     ${ FileToInject = '.\Parent Directory\file.json', InjectAsJsonObject }
  #>
 
 param (
+
     [string] $Path = $PSScriptRoot
 )
 
+# [System.IO.Path]::IsPathFullyQualified() is not supported in PS 5.1 (as opposed to PS 7)
+function IsPathFullyQualified {

```

Comment:
It's a good thing that we have this, then, if it's not supported. I'm wondering though, if this is still good to follow. We're toggling booleans twice just for the sake of this method to be called `IsPathFullyQualified`.

I'll let @pim-simons have a look. I was thinking if it were something in the like of `IsPathRelative` it wouldn't need to to be switched with `-not (-not ( ) )`.